package.json

```diff
@@ -2,37 +2,83 @@
   "name": "@4c/env",
   "version": "1.2.0",
   "description": "Environment variable management and helpers",
-  "main": "env.js",
   "license": "MIT",
   "scripts": {
+    "build:es": "babel src -d es --env-name esm --ignore **/__tests__ --delete-dir-on-start -x .ts",
```

Comment:
es build seems unnecessary here

Reply:
forgive my ignorance, what's the purpose of the ES build at all?

Reply:
it allows scope hoisting in webpack